amazonorders/forms.py

```diff
@@ -244,7 +303,7 @@ class CaptchaForm(AuthForm):
     def __init__(self,
                  config: AmazonOrdersConfig,
                  selector: Optional[str] = None,
-                 error_selector: str = None,
+                 error_selector: Optional[str] = None,
```

Comment:
So the reason I did things like this (you see the one above it has the `Optional`) is because, the default value is `None`, yes, but you as the caller shouldn't pass a `None` in. This is an issue where `mypy` doesn't give us good options ... maybe this is a fine solution, to allow `Optional`, I know `mypy` is more a guide than a hard and fast rule anyway, I just avoided this when I was initially implementing because it felt wrong. But maybe it's fine ... I wouldn't let it block merging the PR, just wondering if you have more thoughts on this.